src/fragments/sdk/storage/ios/getting-started.mdx

```diff
@@ -86,7 +86,3 @@ Run `pod install --repo-update` before you continue.
    ```swift
    import AWSS3
    ```
-
-### Mocking and Local Testing
```

Comment:
Guessing that the staging site is still building? Still seeing it here: https://correct-content-filtering.d1egzztxsxq9xz.amplifyapp.com/swift/sdk/storage/getting-started/#mocking-and-local-testing

Reply:
Yes, just added a note in the description.

Reply:
I see it's gone now!